jim_tcl.txt

```diff
@@ -719,9 +719,11 @@ White space may be used between the operands and operators and
 parentheses; it is ignored by the expression processor.
 Where possible, operands are interpreted as integer values.
 
-Integer values may be specified in decimal (the normal case) or in
-hexadecimal (if the first two characters of the operand are '0x').
-Note that Jim Tcl does *not* treat numbers with leading zeros as octal.
+Integer values may be specified in decimal (the default case that can
+be make explicit by prepending '0d' to the operand) or in binary, octal
+or hexadecimal (if the first two characters of the operand are '0b',
+'0o' or '0x' respectively).  Note that Jim Tcl does *not* treat numbers
+with leading zeros as octal.
```

Comment:
Suggest the following rephrasing:

Integer values are interpreted as decimal, binary, octal or hexadecimal if prepended with '0d', ''0b', '0o' or '0x' respectively. Otherwise they will be interpreted as decimal by default. (Jim Tcl does *not* interpret numbers with leading zeros as octal.)

Reply:
I like this rephrasing.  I'll use it and credit you.